nav2_map_server/src/map_saver/map_saver.cpp

```diff
@@ -66,6 +66,10 @@ MapSaver::on_configure(const rclcpp_lifecycle::State & /*state*/)
   // Make name prefix for services
   const std::string service_prefix = get_name() + std::string("/");
 
+  free_thresh_default_ = get_parameter("free_thresh_default").as_double();
```

Comment:
In the constructor, we need to remove the setting of values too so the declare lines aren't impacting class members (e.g. remove the `X =` )